docs/data-sources/repository/local/cargo_repository.md

```diff
@@ -0,0 +1,29 @@
+---
+subcategory: "Local Repositories"
+---
+
+# Artifactory Local Cargo Repository Data Source
+
+Retrieves a local cargo repository.
+
+## Example Usage
+
+```hcl
+data "artifactory_local_cargo_repository" "terraform-local-test-cargo-repo-basic" {
+  key = "terraform-local-test-cargo-repo-basic"
+}
+```
+
+## Argument Reference
+
+The following argument is supported:
+
+* `key` - (Required) the identity key of the repo.
+
+## Attribute Reference
+
+The following attributes are supported, along with the [common list of attributes for the local repositories](local.md):
+
+* `anonymous_access` - (Optional) Cargo client does not send credentials when performing download and search for crates.
```

Comment:
I merged https://github.com/jfrog/terraform-provider-artifactory/pull/661 yesterday so you should include the new attribute.

Reply:
cdnRedirect right? will do, will do a merge commit from master then go from there

Reply:
See the PR. It's the Cargo's `enabled_sparse_index` attribute.